public/assets/css/main.css

```diff
@@ -0,0 +1,118 @@
+:root {
```

Comment:
Maybe turn this file into a sass (.scss) file for consistency with the rest of the styling

Reply:
done